webExtensionApis/prefs/api.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
+/* global ExtensionAPI:false */
```

Comment:
Not :100: what's happenin' here.
I _was_ getting the following errors:

```
/Users/pdehaan/dev/github/mozilla/shield-studies-addon-utils/webExtensionApis/prefs/api.js
  12:14  error    'Services' is not defined                      no-undef
  14:14  error    'Services' is not defined                      no-undef
  16:14  error    'Services' is not defined                      no-undef
  27:14  error    'Services' is not defined                      no-undef
  29:14  error    'Services' is not defined                      no-undef
  31:14  error    'Services' is not defined                      no-undef
  36:1   warning  Line 36 exceeds the maximum line length of 80  max-len
  37:28  error    'ExtensionAPI' is not defined                  no-undef
```

Turns out the `'Services' is not defined` error was because of some bad `overrides` glob in .eslintrc.js (which was still pointing to ye olde "./src/**" directory, so our `webextensions:true` env wasn't getting set correctly. I'm guessing `ExtensionAPI` is something else/new and not in the webextensions environment global variables yet, or we just need to explicitly set it as a global in each file or in .eslintrc.js root or overrides. 🤷‍♀️

Reply:
I am agnostic on this. Any fix is welcome :)